randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+            "descent with an Oculus and using complex bomb jumps to rise back to the top."
+        ),
+    ],
+    "1530945290": [
+        # Training Chamber Ghost Skip Tip
+        # Replaces Meditation Fountain left sculpture scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;In Training Chamber, one can avoid awakening the Chozo Ghosts "
+            "by climbing architecture near the entrance to Training Chamber Access."
+        ),
+    ],
+    "3721670052": [
+        # Magma Pool Grapple Skip Tip
+        # Replaces Meditation Fountain right sculpture scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;There are several ways to cross Magma Pool without the Grapple Beam.\n\n"
+            "The simplest uses the &main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; and &main-color=#FF3333;"
+            "Gravity Suit&main-color=#89D6FF; to jump off of a raised stone in the lava near the far ledge.\n"
+            "A similar method uses a double bomb jump from the same location, though it requires not having the "
+            "Gravity Suit.\nA harder method uses &main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; to scan-dash"
+            " off of pickups left by the crates.\n\nFinally, it is possible to get across with "
+            "&main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; by L-jumping between the lips of the "
+            "leftmost pillars.\nThis is more difficult than it appears, as the closest pillar is not standable."
+        ),
+    ],
+    "455264308": [
+        # No Grappling/Spidering Thru Damage Tip
+        # Replaces Meditation Fountain panel scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Taking damage will interrupt use of the Spider Ball or Grapple Beam. "
+            "This includes damage from superheated rooms."
+        ),
+    ],
+    "2410883438": [
+        # Just Hold L Tip
+        # Replaces Main Plaza bird image scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Holding &image=SA,3.0,1.0,1.0,46434ED3,34E79314; grants greater "
+            "control over aerial movement, making precise jumps easier."
+        ),
+    ],
+    "389886815": [
+        # Just Tap B Tip
+        # Replaces Observatory left computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;When performing a scan dash or lock-on dash, jump by tapping "
+            "&image=SI,0.70,0.68,833BEE04; for as little time as possible to gain extra height."
+        ),
+    ],
+    "3668253850": [
+        # Item Drop Tip
+        # Replaces Waterfall Cavern hole scan
```

Comment:
This seems like an arbitrary location to put this scan.

Reply:
I wanted to place it early in the game (given a vanilla start), but it's true that it's otherwise arbitrary. I'm open to alternate suggestions.